### DISCUSSION TITLE
[Suggestion] /cmi unchangeable

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#1371] [Feature Requests]
Sometimes we have custom items, with /cmi enchant, /cmi itemname, lore, etc. And we dont want players to enchant these, merge them, or repair them. 

Would it be possible to get a flag or something that we can set with holding the item in our hand and typing: /cmi unchangable or something logical like that?

So when a player adds an event item in the anvil or whatever, and adds a mending book to it, that they get the msg 'this item cannot be changed, sorry'?

**Comment by PreciseOXT** [comment]
This is something I'd really like to see implemented as well.
Currently, we put infinity on OP Crate Prize tools/armour so they can't have mending, It still allows players to repair the items with diamonds a number of times though.
An unchangeable option would be ideal.
